msal/src/main/java/com/microsoft/identity/client/TokenParameters.java

```diff
@@ -154,13 +159,34 @@ public String getCorrelationId() {
         return mCorrelationId;
     }
 
+    /**
+     * If you've been instructed to pass additional query string parameters to the authorization endpoint.  You can get these here.
+     * Otherwise... would recommend not touching.
+     *
+     * @return
+     */
+    public List<Pair<String, String>> getExtraQueryStringParameters() {
+        return mExtraQueryStringParameters;
+    }
+
+    /**
+     * If you have been instructed that the client requires extra parameters, supply them here.
```

Comment:
Can we use the same word (parameters vs options)? Also can you add some examples of what might these extra parameters be?

Reply:
Yes.  Probably options.

In this case, the only magic parameter that we will be dealing with would be triggers for special or potentially experimental client side behavior.